packages/cmr-client/src/getConcept.ts

```diff
@@ -3,35 +3,53 @@ import Logger from '@cumulus/logger';
 
 const log = new Logger({ sender: 'cmr-client' });
 
-/**
- * Get the CMR JSON metadata from the cmrLink
- *
- * @param {string} conceptLink - link to concept in CMR
- * @param {Object} headers - the CMR headers
- * @returns {Object} - metadata as a JS object, null if not
- * found
- */
-async function getConceptMetadata(
+async function getGranuleConceptMetadata(
   conceptLink: string,
-  headers: Headers
+  headers: Headers,
+  type: string
 ): Promise<unknown> {
   let response;
-
   try {
     response = await got.get(conceptLink, { headers });
   } catch (error) {
     log.error(`Error getting concept metadata from ${conceptLink}`, error);
-    return null;
+    return undefined;
   }
 
   if (response.statusCode !== 200) {
```

Comment:
Handle 404 null return for 'granuleConceptSearch' type

Reply:
Does this also need to be done for the `conceptExists` function below?

Reply:
It is - it handles a 404 by setting response to error.response, then false when the status code (404) is !==200 on L150

Reply:
Oh, I see. Right.